PlanPluginBridge/src/main/java/com/djrapitops/pluginbridge/plan/luckperms/LuckPermsData.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Licence is provided in the jar as license.yml also here:
+ * https://github.com/Rsl1122/Plan-PlayerAnalytics/blob/master/Plan/src/main/resources/license.yml
+ */
+package com.djrapitops.pluginbridge.plan.luckperms;
+
+import com.djrapitops.plan.data.element.AnalysisContainer;
+import com.djrapitops.plan.data.element.InspectContainer;
+import com.djrapitops.plan.data.element.TableContainer;
+import com.djrapitops.plan.data.plugin.ContainerSize;
+import com.djrapitops.plan.data.plugin.PluginData;
+import com.djrapitops.plan.utilities.html.icon.Color;
+import com.djrapitops.plan.utilities.html.icon.Family;
+import com.djrapitops.plan.utilities.html.icon.Icon;
+import com.djrapitops.plan.utilities.html.icon.Icons;
+import java.util.Collection;
+import java.util.List;
+import java.util.OptionalInt;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import me.lucko.luckperms.api.*;
+import me.lucko.luckperms.api.caching.MetaData;
+
+public class LuckPermsData extends PluginData {
+    private LuckPermsApi api;
+
+    public LuckPermsData(LuckPermsApi api) {
+        super(ContainerSize.THIRD, "LuckPerms");
+        setPluginIcon(Icon.called("exclamation-triangle").of(Family.SOLID).build());
```

Comment:
`.of(Family.SOLID)` is not necessary. (Solid is default)

This icon has no color (It will appear as black)

[List of available color names](https://gurayyarar.github.io/AdminBSBMaterialDesign/pages/ui/colors.html) (Lowercase)